### PR TITLE
Initial commit of AWX

### PR DESCRIPTION
**Pull request by dasJ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Ubuntu
+
+## Role Variables
+
+| Name                | Default/Required    | Description                                      |
+|---------------------|:-------------------:|--------------------------------------------------|
+| `awx_arch`          | `amd64`             | Architecture of the target CPU                   |
+| `awx_postgres_data` | `/var/lib/postgres` | Location of the data of the PostgreSQL container |
+| `awx_tunables`      |                     | Dict of all installer tunables                   |
```
Comment (SlothOfAnarchy) [inline README.md]: Please explain or link to the tunables.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## Requirements
+
+Ubuntu
+
+## Role Variables
+
+| Name                | Default/Required    | Description                                      |
+|---------------------|:-------------------:|--------------------------------------------------|
+| `awx_arch`          | `amd64`             | Architecture of the target CPU                   |
+| `awx_postgres_data` | `/var/lib/postgres` | Location of the data of the PostgreSQL container |
```
Comment (SlothOfAnarchy) [inline README.md]: This var is not used. Did you intend to add our postgres role as a dependency and somehow use it here?

File: tasks/awx.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- name: Create directory for AWX
+  file:
+    path: "{{ awx_home }}"
+    state: directory
+    owner: root
+    group: root
+    mode: 0750
+
+- name: Clone AWX
+  git:
```
Comment (SlothOfAnarchy) [inline tasks/awx.yml:11]: I suggest checking out a specific version. That way we avoid introducing breaking changes.

File: tasks/docker.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+- name: Add Docker APT key
+  apt_key:
+    url: https://download.docker.com/linux/ubuntu/gpg
+
+- name: Install required packages for Docker
```
Comment (SlothOfAnarchy) [inline tasks/docker.yml]: Can't you install all packages in one task, after the repository has been added?

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: tasks/awx.yml (hunk 3 above)
Comment (dasJ) [reply tasks/awx.yml:11]: I cannot find anything I can use

File: README.md (hunk 2 above)
Comment (dasJ) [reply README.md]: No, AWX comes with a postgresql container

File: README.md (hunk 1 above)
Comment (dasJ) [reply README.md]: There is currently no documentation on them :/

File: README.md (hunk 2 above)
Comment (SlothOfAnarchy) [reply README.md]: Still, I can't see where the var is used?

File: README.md (hunk 1 above)
Comment (SlothOfAnarchy) [reply README.md]: Somehow you found them, you could link to the source of that or explain some of them.

**Comment by dasJ** [comment]
@SlothOfAnarchy Pls rereview

File: tasks/install.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
+- name: Add Docker APT key
+  apt_key:
+    url: https://download.docker.com/linux/ubuntu/gpg
+
+- name: Update repository cache
```
Comment (SlothOfAnarchy) [inline tasks/install.yml:6]: Shouldn't this be done after new repos have been added?

File: tasks/install.yml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  apt:
+    update_cache: true
+  changed_when: false
+
+# Required for the Docker repo
+- name: Add HTTPS support to APT
+  apt:
+    name: apt-transport-https
```
Comment (SlothOfAnarchy) [inline tasks/install.yml:14]: Please move this to the iterative install task below.

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**